tests/foreman/cli/test_rhcloud_inventory.py

```diff
@@ -139,3 +132,79 @@ def test_positive_sync_inventory_status():
 
     :CaseLevel: System
     """
+
+
+@pytest.mark.stubbed
+def test_rhcloud_inventory_bz_1962694():
+    """Verify that if organization had more hosts than specified by max_org_size variable
+        then report won't be uploaded.
+
+    :id: 7dd964c3-fde8-4335-ab13-02329119d7f6
+
+    :Steps:
+
+        1. Register few content hosts with satellite.
+        2. Change value of max_org_size for testing purpose(See BZ#1962694#c2).
+        3. Start report generation and upload using
+            ForemanInventoryUpload::Async::GenerateAllReportsJob.perform_now
+
+    :expectedresults: If organization had more hosts than specified by max_org_size variable
+        then report won't be uploaded.
+
+    :CaseImportance: Low
+
+    :BZ: 1962694
+
+    :CaseAutomation: ManualOnly
```

Comment:
Is there no way to automate any of these test cases, as indicated by `ManualOnly`?

Reply:
They are hard to automate, some are destructive in nature and not worth automating.

Reply:
Ack, double checking this is what you wanted, instead of `NotAutomated`